Dockerfile

```diff
@@ -0,0 +1,32 @@
+
+FROM ubuntu:latest
+
+ARG HOMEBREW_GITHUB_API_TOKEN=""
+
+RUN apt-get update && \
+    apt-get install build-essential curl file git ruby-full locales sudo snapd --no-install-recommends -y && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN localedef -i en_US -f UTF-8 en_US.UTF-8
+
+RUN useradd -m -s /bin/bash linuxbrew && \
+    echo 'linuxbrew ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
+
+USER linuxbrew
+RUN sh -c "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install.sh)"
+
+USER root
+ENV PATH="/home/linuxbrew/.linuxbrew/bin:${PATH}"
+
+USER linuxbrew
+ENV OSTYPE darwin23.0
+
+RUN brew install gum kubectl helm && brew cleanup --prune=all
```

Comment:
@grapple-solutions  do we need this explicit step? since all of the pre-reqs installation issues are fixed now.

Reply:
Hi @zahid-iqbal-alpha 
Thanks for the feedback.
The docker image should from my point of view contain all the absolutely required tools out of the box in order to improve performance at execution time.
And without kubectl and helm not one of the grapple related functions work.
I explicitly excluded civo cli and other tools from the installation because they are only required for certain use cases.
but the mandatory stuff I have put into the image.

I could even imagine in future to build different tags for certain use cases (one tag for civo usage, another tag for aws usage - to have all the needed tools installed for faster execution of the commands in the container)

And the installation process in the CLI was designed mainly for the installation on "clients" (like on a mac book, etc)